<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!-- Move Base Node -->
  <node name="move_base" output="screen" pkg="move_base" type="move_base">
    <rosparam command="load" file="$(find navigation)/config/local_costmap_params.yaml"></rosparam>
    <rosparam command="load" file="$(find navigation)/config/global_costmap_params.yaml"></rosparam>
    <rosparam command="load" file="$(find navigation)/config/base_global_planner.yaml"></rosparam>
    <param name="base_global_planner" value="navfn/NavfnROS"></param>
    <!-- <rosparam file="$(find navigation)/config/base_local_planner.yaml" command="load" /> -->
    <!-- <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" /> -->
    <rosparam command="load" file="$(find navigation)/config/teb_local_planner_params.yaml"></rosparam>
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"></param>
    <remap from="cmd_vel" to="/cmd_vel"></remap>
    <remap from="scan" to="/ouster/scan2"></remap>
    <remap from="odom" to="/robot/dlo/odom_node/odom"></remap>
    <remap from="map" to="/global_map"></remap>
  </node>
</launch>
